docs/basic_training/intro.es.md

```diff
@@ -0,0 +1,274 @@
+10. Fin del primer bloque de proceso.
+11. Comienza el segundo bloque de proceso, definido como `MAYUSCULAS`.
+12. Declaración de entrada para el `proceso` `MAYUSCULAS`.
+13. Le dice al `proceso` que espere un archivo(s) de `entrada` (`ruta`; es decir, segmento_aa y segmento_ab), que asignamos a la variable 'y'.
+14. Declaración de salida para el proceso `MAYUSCULAS`.
+15. Le dice al proceso que espere la salida como salida estándar (stdout) y envía esta salida como un canal.
+16. Tres comillas dobles inician y finalizan el bloque de código para ejecutar este `proceso`.
+    Dentro del bloque hay una secuencia de comandos para leer archivos (cat) usando la variable de entrada '$y', luego realizar la conversión a mayúsculas y generar la salida estándar.
+17. Fin del segundo bloque `proceso`.
+18. Inicio del alcance del flujo de trabajo donde se puede llamar a cada proceso.
+19. Ejecute el `proceso` `SEPARALETRAS` en `saludo_ch` (también conocido como canal de saludo), y almacene la salida en el canal `letras_ch`.
+20. Ejecute el `proceso` `MAYUSCULAS` en el canal de letras `letras_ch`, que se aplana usando el operador `.flatten()`. Esto transforma el canal de entrada de tal manera que cada elemento es un elemento separado. Almacenamos la salida en el canal `resultados_ch`.
+21. El resultado final (en el canal `resultados_ch`) se imprime en la pantalla usando el operador `view` (junto al nombre del canal).
+22. Fin del flujo de trabajo.
+
+El operador `.flatten()` se usa aquí para dividir los dos archivos en dos elementos separados para pasar por el siguiente proceso (de lo contrario, serían tratados como un solo elemento).
+
+### En la práctica
+
+Ahora copie el ejemplo anterior en su editor de texto favorito y guárdelo en un archivo llamado `hello.nf`.
+
+!!! advertencia
+
+    Para el tutorial de Gitpod, asegúrese de estar en la carpeta llamada `nf-training`
+
+Ejecute el script ingresando el siguiente comando en su terminal:
+
+```bash
+nextflow run hola.nf
+```
+
+El resultado será similar al texto que se muestra a continuación:
+
+```linenums="1"
+N E X T F L O W  ~  version 22.04.5
+Launching `hola.nf` [gigantic_poitras] DSL2 - revision: 197a0e289a
+executor >  local (3)
+[c8/c36893] process > SEPARALETRAS (1)   [100%] 1 of 1 ✔
+[1a/3c54ed] process > MAYUSCULAS (2) [100%] 2 of 2 ✔
+MUNDO!
+¡HOLA
+```
+
+La salida estándar muestra (línea por línea):
+
+1. La versión de Nextflow que se ejecutó.
+2. Los nombres de la secuencia de comandos y la versión.
+3. El ejecutor utilizado (en el caso anterior: local).
+4. El primer `proceso` se ejecuta una vez. La línea comienza con un valor hexadecimal único (consulte el CONSEJO a continuación) y finaliza con el porcentaje y la información de finalización del trabajo.
+5. El segundo proceso se ejecuta dos veces (una vez para segmento_aa y otra para segmento_ab).
+6. Se imprime la cadena de resultado de stdout.
```

Comment:
Similar to mentioned above, I would use **string** instead of cadena de resultado